deployment/run_server

```diff
@@ -1,10 +1,25 @@
 #!/bin/bash -xe
 
 fc-cache -v /usr/share/fonts/user
+set -e
+
+BUILD_DIR=/var/www/image-exporter/build
+URL_REGEXP='(https?|ftp|file)://[-A-Za-z0-9\+&@#/%?=~_|!:,.;]*[-A-Za-z0-9\+&@#/%=~_|]'
```

Comment:
This is unneeded now. Just pass an environment variable that, if set, contains a URL. That will be significantly more maintanable than depending on a regular expression.

Reply:
In other words there should be two modes:
* Retrieve plotly.js from a URL and pass that. In this case we download and add the argument.
* The caller specifies the `--plotlyJS` argument.